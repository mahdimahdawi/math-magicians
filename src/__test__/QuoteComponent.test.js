import { render, screen } from '@testing-library/react';
import Quote from '../components/quote';

describe('Testing Quote Component', () => {
  test('testing quote', () => {
    render(<Quote />);

    // eslint-disable-next-line testing-library/await-async-query
    expect(screen.findByText('Mathematics is not about number, Equations, computations, or Algorithms: it is about Understanding. -William Paul Thurston')).toMatchSnapshot();
  });
});