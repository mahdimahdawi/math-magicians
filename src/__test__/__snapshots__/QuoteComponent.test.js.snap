// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Quote Component testing quote 1`] = `Promise {}`;
